packages/@react-aria/ssr/src/SSRProvider.tsx

```diff
@@ -93,7 +93,7 @@ export function SSRProvider(props: SSRProviderProps): JSX.Element {
   return <LegacySSRProvider {...props} />;
 }
 
-let canUseDOM = Boolean(
+export let canUseDOM = Boolean(
```

Comment:
unfortunately, useIsSSR doesn't work in all cases, I'm not sure why.
this check works for our use case here, if we don't want to export it, i could copy it over, it's quite small

Reply:
ya copy it